wiki/Glossary/fr.md

```diff
@@ -0,0 +1,999 @@
+     - en bas à gauche en osu!standard
+     - sur les tambours en osu!taiko
+     - au dessus du fruit catcher en osu!catch
+     - au centre du niveau en osu!mania (la hauteur verticale varie en fonction du skin utilisé)
+
+### Comboburst
+
+Le comboburst est une collection d'éléments/images qui _"burst"_ depuis la gauche ou la droite de l'écran (ou la droite du niveau en mode osu!mania) après avoir obtenu un combo de 30, 60, 100, (multiples de 50, après 100).
+
+### Combo colour
+
+La combo colour est une couleur qui sépare un combo d'un autre.
+
+En osu!standard, tous les combos ont une couleur.
+Un nouveau combo démarre quand la couleur de l'objet à frapper change.
+Les couleurs se répèteront indéfiniment en fonction du nombre de couleurs défini par le mappeur (ou de combien de couleurs sont définies dans le skin.ini du skin personnalisé).
+Il y'a un minimum de 2 et un maximum de 8 couleurs de combo.
+
+En osu!catch, chaque objet est coloré dans l'ordre donné.
+Les droplets et fruits de fin de sliders auront toujours la même couleur que le fruit de départ.
+Cependant, les bananes auront toujours une teinte jaune.
+
+A l'édition d'une beatmap, il est possible de changer les couleurs et leur ordre dans la section "Song Setup".
+
+Pour les skins, les propriétés pour les couleurs personnalisées sont situées dans la section `[Colours]` du fichier skin.ini.
+Cependant, veuillez noter que les couleurs de combo définies dans le skin sont utilisées que si le mappeur **ne les définit pas** pour une beatmap.
+
+### Combo fire
+
+![R.I.P. combo fire](img/combo-fire.jpg)
+
+Le combo fire était une fonctionnalité dans osu! qui affichait une flamme jaune après avoir obtenu un combo de 30.
+Si le joueur obtenait un combo de 500, la couleur du combo fire passait du jaune au bleu.
+Le combo fire s'éteignait de lui-même si le combo était cassé, mais pouvait toujours être redéclanché.
+
+Cette fonctionnalité a été par la suite désactivée à cause de problèmes de performances.
+
+### Cookie
+
+![](img/Intro_static.jpg "The cookie is typically used to describe the circular osu!logo which appears at the very start of the game, and refers to its shape.")
+
+Le cookie est un autre terme désignant le "osu!logo", qui apparaît plusieurs fois dans le client osu!.
+Le cookie est présent dans le menu principal, en mode solo, et précédemment dans les vieux rapports de crash ([voir un exemple](img/Pippi_corruption.jpg)).
+
+## D
+
+### Deathstreams
+
+Un deathstream est un long flux ou les objets se suivent les uns les autres en se succédant de manière rapide et/ou en formant des patterns assez confus.
+Ils amènent généralement le joueur à perdre sa "stamina" de manière rapide, et potentiellement à échouer la beatmap.
+
+Il est fortement suggéré d'utiliser les deathstreams uniquement lorsque la beatmap l'insinue et/ou qu'elle a un BPM élevé.
+Ce principe **ne devrait pas** être utilisé pour faire des overmaps!
+
+### Etoiles (terme original : Difficulty stars)
```

Comment:
ditto

if you are going to include the original term, please put it underneath instead

```
### Etoiles

_Terme original : Difficulty stars_
```